File: bialydunajec-adminpanel-frontend/projects/bialydunajec-admin/src/app/shared/service/rest/dto/address.dto.ts
export class AddressDto {
  street?: string;
  homeNumber?: string = null;
  city: string;
  postalCode?: string = null;


  constructor(street: string, homeNumber: string, city: string, postalCode: string) {
    this.street = street;
    this.homeNumber = homeNumber;
    this.city = city;
    this.postalCode = postalCode;
  }
}

File: backend/bialydunajec-ddd/dto/src/main/kotlin/org/bialydunajec/ddd/base/dto/AddressDto.kt
package org.bialydunajec.ddd.base.dto


data class AddressDto(
        //@field:NullOrNotBlank
        val street: String? = null,

        //@field:NullOrNotBlank
        val homeNumber: String? = null,

        //@field:NullOrNotBlank
        val city: String? = null,

        //@field:NullOrNotBlank
        val postalCode: String? = null
)